src/main/java/gmbs/model/Cars.java

```diff
@@ -0,0 +1,56 @@
+package gmbs.model;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import gmbs.util.RandomGenerator;
+
+import static java.lang.System.lineSeparator;
+
+public class Cars {
+    private static final String DELIMITER = ",";
+    private static final int MIN = 0;
+    private static final int MAX = 9;
+    private static final String DUPLICATED_NAME_MESSAGE = "[ERROR] 중복된 이름이 있습니다";
+    private final List<Car> cars = new ArrayList<>();
+
+    public Cars(String carNames) {
+
+        String[] carNameArray = reduceBlank(carNames).split(DELIMITER);
+        validateDuplicatedName(carNameArray);
+        for (String carName : carNameArray) {
+            cars.add(new Car(carName));
+        }
+    }
```

Comment:
```suggestion
public class Cars {
    private static final String DELIMITER = ",";
    private static final int MIN = 0;
    private static final int MAX = 9;
    private static final String DUPLICATED_NAME_MESSAGE = "[ERROR] 중복된 이름이 있습니다";
    private final List<Car> cars;

    public Cars(String carNames) {
        String[] carNameArray = reduceBlank(carNames).split(DELIMITER);
        List<Car> cars = new ArrayList<>();
        validateDuplicatedName(carNameArray);
        for (String carName : carNameArray) {
            cars.add(new Car(carName));
        }
        this.cars = cars;
    }
```
필드에 선언해두고 add하는 것도 방법이지만, 생성자에서 ArrayList를 생성해서 할당하는건 어떨까요?